src/cursive.rs

```diff
@@ -81,7 +97,7 @@ impl Cursive {
     /// regularly check for messages.
     ///
     /// [`set_fps`]: #method.set_fps
-    pub fn cb_sink(&self) -> &mpsc::Sender<Box<Fn(&mut Cursive) + Send>> {
+    pub fn cb_sink(&self) -> &mpsc::Sender<Box<CbFunc>> {
```

Comment:
I think a small example code here, or a link to an example, would help users understand that they can just send regular closures here.